ideascube/serveradmin/tests/data/a-video.mp4

```diff
@@ -0,0 +1 @@
+aaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Commenting about the images: don't we already have them for the tests somewhere?